app-accessibility/at-spi2-core/at-spi2-core-2.47.90.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson-multilib systemd virtualx xdg
+
+DESCRIPTION="D-Bus accessibility specifications and registration daemon"
+HOMEPAGE="https://wiki.gnome.org/Accessibility https://gitlab.gnome.org/GNOME/at-spi2-core"
+
+LICENSE="LGPL-2.1+"
+SLOT="2"
+IUSE="X gtk-doc +introspection"
+REQUIRED_USE="gtk-doc? ( X )"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+
+RDEPEND="
+	>=sys-apps/dbus-1.5[${MULTILIB_USEDEP}]
+	>=dev-libs/glib-2.67.4:2[${MULTILIB_USEDEP}]
+	>=dev-libs/libxml2-2.9.1:2[${MULTILIB_USEDEP}]
+	introspection? ( >=dev-libs/gobject-introspection-1.54.0:= )
+	X? (
+		x11-libs/libX11[${MULTILIB_USEDEP}]
+		x11-libs/libXtst[${MULTILIB_USEDEP}]
+		x11-libs/libXi[${MULTILIB_USEDEP}]
+	)
+
+	!<dev-libs/atk-2.46.0
+	!<app-accessibility/at-spi2-atk-2.46.0
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/glib-utils
+	gtk-doc? (
+		>=dev-util/gi-docgen-2021.1
```

Comment:
Usually packages use gi-docgen XOR gtk-doc, so I suspect it no longer needs gtk-doc. (See upstream commit https://gitlab.gnome.org/GNOME/at-spi2-core/-/commit/837c050e48ea73f6dcf5ef9ff7120bc7da8b38ad)
I think the dependency on `app-text/docbook-xml-dtd` can also be dropped.

Upstream commit https://gitlab.gnome.org/GNOME/at-spi2-core/-/commit/7a0c7f420dca9793c5bb44e0406de3eb5d682284 looks like it added a dependency on `dev-python/sphinx` and `dev-util/gdbus-codegen` as well, for `USE=gtk-doc`.